client/src/components/ProfileCard/ProfileCard.css

```diff
@@ -1,37 +1,34 @@
 .profileCard_card{
-    margin-top: 10px;
-    margin-bottom: 10px;
-    border: solid 2px red;
+    margin-top: 1%;
```

Comment:
For this project anyway, try using em for unit of measurements. That way, it'll be easier to keep things looking uniform throughout the project.